src/library/Uno.Themes.WinUI.Markup/ResourceKeyDefinitionAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Uno.Themes.WinUI.Markup;
+
+// temporary workaround for ResourceKeyDefinition not applicable on field members.
```

Comment:
Should we open an issue in the markup repo to track changing this

Reply:
It should be fixed on markup repo before this is merged and then this can be removed I assume

Reply:
https://github.com/unoplatform/uno.csharpmarkup/pull/452

Reply:
The linked PR has been merged. Keep this unresolved until this class has been removed

Reply:
@nickrandolph Just to point out, this PR is being replaced by #1247.
And, this issue has been fixed over there:  a04959ca